Competition2017/src/competition/subsystems/shoot_fuel/StopFeedingAndCollectionCommandGroup.java

```diff
@@ -0,0 +1,21 @@
+package competition.subsystems.shoot_fuel;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.agitator.AgitatorsManagerSubsystem;
+import competition.subsystems.collector.commands.StopCollectorCommand;
+import competition.subsystems.shooter_belt.ShooterBeltsManagerSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+
+public class StopFeedingAndCollectionCommandGroup extends StopAllShootingCommandGroup {
```

Comment:
Consider inverting this relationship so the `StopAllShootingCommandGroup` extends `StopFeedingAndCollectionCommandGroup` by adding the shooterwheels into the mix?